lucien-ios/AddNewComicBook/AddComicViewController.swift

```diff
@@ -237,8 +251,46 @@ final class AddComicViewController: UIViewController, UIPickerViewDelegate, UIPi
     }
 
     func textFieldDidEndEditing(_ textField: UITextField) {
+        activeField = nil
         guard let textFieldSubLayers = textField.layer.sublayers else { return }
         let bottomBorder = textFieldSubLayers[0] as CALayer
         bottomBorder.backgroundColor = LucienTheme.silver.cgColor
     }
+
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+        if let nextTextField = textField.superview?.superview?.viewWithTag(textField.tag + 1) as? UITextField {
+            nextTextField.becomeFirstResponder()
+        } else {
+            textField.resignFirstResponder()
+        }
+        return false
+    }
+
+    // MARK: - Keyboard Methods
+
+    func registerForKeyboardNotifications() {
+        NotificationCenter.default.addObserver(self, selector: #selector(AddComicViewController.keyboardWasShown), name: NSNotification.Name.UIKeyboardWillShow, object: nil)
+    }
+
+    func deregisterFromKeyboardNotifications() {
+        NotificationCenter.default.removeObserver(self, name: NSNotification.Name.UIKeyboardWillShow, object: nil)
+    }
+
+    @objc func keyboardWasShown(notification: NSNotification) {
+        guard let info = notification.userInfo,
+              let keyboardSize = (info[UIKeyboardFrameBeginUserInfoKey] as? NSValue)?.cgRectValue.size else {
+                return
+        }
+        scrollView.isScrollEnabled = true
+        let contentInsets = UIEdgeInsets(top: 0.0, left: 0.0, bottom: keyboardSize.height + LucienConstants.keyBoardHieghtPadding, right: 0.0)
+        scrollView.contentInset = contentInsets
+        scrollView.scrollIndicatorInsets = contentInsets
+        var aRect = view.frame
```

Comment:
This naming isn't very clear. What is `aRect`? If it's the view's frame, name it `viewFrame`